docs/docs/spec.md

```diff
@@ -340,9 +340,12 @@ To determine whether a type usage appears in a null-marked scope:
 Iterate over all the declarations that enclose the type usage, starting from the
 innermost.
 
-"Enclosing" is defined as follows: Class members are enclosed by classes, which
-may be enclosed by other class members or classes. Top-level classes are
-enclosed by packages, which may be enclosed by modules.
+"Enclosing" is defined as follows:
+
+-   Class members are enclosed by classes.
```

Comment:
Should we make these singular to avoid ambiguity about how many (directly) enclosing things there are?

"Each class member is enclosed by a class", etc.

Reply:
Yes, that sounds good to me.